saviSimulation/savi/StateSynchronization/CameraPerception.java

```diff
@@ -6,12 +6,12 @@
 
 public class CameraPerception extends Perception {
 
-	public CameraPerception(String perceptionName, double timeStamp, double azumuth, double elevation, double range, double radius) {
-		this(perceptionName, new String("unknown"), timeStamp, azumuth, elevation, range, radius);
+	public CameraPerception(String perceptionName, int unique_id, double timeStamp, double azumuth, double elevation, double range, double radius) {
```

Comment:
can you change azumuth to azimuth (I know you didn't introduce that, but while I'm reviewing...)

Reply:
Fixed the spelling!